development-guide.md

```diff
@@ -80,14 +80,37 @@ The following are hard requirements to be able to deploy partner code to Fivetra
 - Do not log sensitive data. Ensure only necessary information is kept in logs, and never log any sensitive data. Such data may include credentials (passwords, tokens, keys, etc.), customer data, payment information, or PII.
 - Encrypt HTTP requests. Entities like URLs, URL parameters, and query parameters are always encrypted for logging, and customer approval is needed to decrypt and examine them.
 
+## Setup Form Guidelines
+- Keep the form clear and concise, only requesting essential information for successful connector setup.
+- Use clear and descriptive labels for each form field. Avoid technical jargon if possible.
+- Organize the fields in a logical order that reflects the setup process.
 
-## Connector guidelines
+### RPC Calls
+#### ConfigurationForm
+This operation retrieves all the setup form fields and tests information. You can provide various parameters for the fields to enhance the user experience, such as descriptions, optional fields, and more.
+
+#### Test
+The previous RPC call retrieves the tests that need to be executed during connection setup. This operation then invokes the test with the customer's credentials as parameters. Finally, it should return a success or failure indication for the test execution.
+
+### Supported setup form fields 
+- Text Field: A standard text input field for user text entry.
```

Comment:
Can we add the Text fields available for this:
You provide a 'title' displayed above the field and can optionally provide a 'description' upto x characters long which is displayed below the field and can help explain what should be completed.

Reply:
Added